be/tx.h

```diff
@@ -342,6 +342,12 @@ struct m0_be_tx {
 	 * transaction reaches GROUPED state.
 	 */
 	uint64_t               t_lsn;
+	/**
+	 * No BE transaction with lsn less than this is going to be recovered by
+	 * BE recovery.
+	 * @see m0_be_log_record::lgr_log_header_discarded.
+	 */
+	m0_bindex_t            t_lsn_discarded;
```

Comment:
Is there any changes required for the BE recovery code?
Also "beck" is a utility to do offline BE checking and recovery, I think.
Does "beck" require some changes accordingly?

Reply:
BE data corruption recovery code (to not to mix with BE recovery code in be/recovery.[ch]) inside `m0beck` doesn't run as part of the cluster. FDMI records are sent from one Motr process to another (possibly the same) Motr process inside a Motr cluster. It means that it doesn't make any sense to add anything FDMI-related to `m0beck`.